tools/agent_tool/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
Add a minimum Readme.md with the instructions to build the package (only vcs import and colcon --packages-up-to) and a pair of examples on how to run it (one with help, one with client, one with server for example)